release-team/role-handbooks/patch-release-manager/README.md

```diff
@@ -225,9 +312,32 @@ The freeze serves several purposes:
 1.  It allows slow test jobs like "serial", which has a period of many hours,
     to run several times at `HEAD` to ensure they pass consistently.
 
-On the day before the planned release, run a mock build with `anago` to make
-sure the tooling is ready.
-If the mock goes well and the tests are healthy, run the real cut the next day.
+On the day before the planned release, run a mock build with `gcbmgr`
+to make sure the tooling is ready as per the [Branch Manager
+Handbook](/release-team/role-handbooks/patch-release-manager/README.md).
+Also give the Google Debs/RPMs build staff notification that their
+help will be needed the next day.  Once you've done the mock build, you can
+also do a mock release and notify, sending the release notification email
+to yourself to visual confirm its content:
+
+```bash
+~$ cd src/k8s.io/release
+~$ /release-notify v1.13.2 --nomock --mailto=you@somewhere.com
+```
+
+If the mock build and release goes well and CI tests show the branch
+is healthy, run the real cut the next day by repeating the build
+and release process without the `--nomock` argument in the commands.
+Collaborate with the Google Debs/RPMs build staff to insure the
+packages are successfully published.  Then announce the release:
+
+```bash
+~$ cd src/k8s.io/release
+~$ /release-notify v1.13.2
+```
+
+This will automatically send the formatted announcement to the
+kubernetes-dev mailing list.
```

Comment:
Will it also mail to kubernetes-dev-announce?

Reply:
Ah it is kubernetes-dev and kubernetes-announce.  We manually send the pre-notification to kubernetes-dev and bcc kubernetes-dev-announce.

Reply:
Right, kubernetes-announce. I think it's worth mentioning where the email is supposed to go.

Looking at the command, did you mean `$ ./release-notify v1.13.2 --no-mock`?